### DISCUSSION TITLE
Understanding Prometheus Scraping Config

### DISCUSSION THREAD

**Discussion by tshaiman** [kedacore/charts#396] [Q&A]
I would like to tell Prometheus to scrape Keda , but my Prometheus is not Prom-Operator so I can't use Pod Monitor CRD..

on Keda-metrics I see the following 
```
prometheus.io/scrape: "true"
        prometheus.io/port: {{ .Values.prometheus.metricServer.port | quote }}
        prometheus.io/path: {{ .Values.prometheus.metricServer.path }}
```
while this annotation is missing from Keda Operator  which looks like this 

![image](https://user-images.githubusercontent.com/512709/218787509-7b15e14a-bb53-4b11-bb2c-f841b757fc56.png)

so it does have some refernce to the Prometheus scraping capabilities but without the annotaton.
what is the reason for that ?

**Comment by tomkerkhove** [comment]
I think @JorTurFer can confirm but I don't think there is a specific reason. If prometheus is enabled, then the bind address should be happening and this is a bug

**Comment by JorTurFer** [comment]
I think that we should support also the option to set the port using just prometheus (IDK why only if supported for PodMonitor). 
I have checked and it has a default value `8080`, so you could just scrape that port and it should work 
`pflag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")`

Are you willing to contribute with the change for supporting also the port configuration for prometheus?
